### PR TITLE
AvatarView added clipping for the different StrokeShape's.

### PR DESCRIPTION
**Pull request by GeorgeLeithead**
### Description of Change ###

 
In the HandlePropertChanged method, for each StrokeShape we apply the appropriate clip (Polyline, Ellipse, Path, Polygon, Rectangle, RoundRectangle) to the image.
Added additional samples to the `AvatarViewRatingPage` to demonstrate issue, verify any fix, and provide additional samples.

 ### Linked Issues ###
 

#510 

 - Fixes #

 ### PR Checklist ###
 
 - [x] Has a linked Issue, and the Issue has been `approved`(bug) or `Championed` (feature/proposal)
 - [ ] Has tests (if omitted, state reason in description)
 - [x] Has samples (if omitted, state reason in description)
 - [ ] Rebased on top of `main` at time of PR
 - [ ] Changes adhere to [coding standard](https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#contributing-code---best-practices)
 - [ ] Documentation created or updated: https://github.com/MicrosoftDocs/CommunityToolkit/pull/XYZ 

 ### Additional information ###

### CODE REVIEW

File: src/CommunityToolkit.Maui/Views/AvatarView.shared.cs
Comment (VladislavAntonyuk) [inline src/CommunityToolkit.Maui/Views/AvatarView.shared.cs:365]: I think you can use switch expression

**Comment by brminnick** [comment]
Hey @GeorgeLeithead! Could you send me your Discord username? Feel free to DM me on Twitter if you don't feel comfortable sharing it here.

We'd like to spin up a channel in our Discord server with you and us core maintainers so that we can answer quick + easy questions for ya!

**Comment by GeorgeLeithead** [comment]
I'm not on Discord. I'll have to set it up.

File: src/CommunityToolkit.Maui/Views/AvatarView.shared.cs
Comment (GeorgeLeithead) [reply src/CommunityToolkit.Maui/Views/AvatarView.shared.cs:365]: I did have it, but went with the switch whilst I has working thru the problem. I'll try and remember to convert when done.

**Comment by GeorgeLeithead** [comment]
> Thanks @GeorgeLeithead! Could you send me your Discord username? Feel free to DM me on Twitter if you don't feel comfortable sharing it here.
> 
> We'd like to spin up a channel in our Discord server with you and us core maintainers so that we can answer quick + easy questions for ya!

@brminnick I have no problem sharing my Discord user name.  It's: GeorgeLeithead#2089

**Comment by pictos** [comment]
@GeorgeLeithead please join this server https://discord.gg/JUdFDHGz, then you can press ctrl + T and type `xamarin-community` it will suggest our channel there

**Comment by GeorgeLeithead** [comment]
Most of the clipping is good, but Windows is proving to be a problem. As such, please don't put it into production until either resolved, or deemed acceptable to fix later.

**Comment by brminnick** [comment]
Adding the `do not merge` tag per George’s request
> Most of the clipping is good, but Windows is proving to be a problem. As such, please don't put it into production until either resolved, or deemed acceptable to fix later.

**Comment by brminnick** [comment]
Hey Gang! Just checking in - are we good to merge this PR today or tomorrow (Wednesday or Thursday)?

We plan on pushing a new release to NuGet on Friday and would love to include this PR! 

If it needs more time, no worries! We can add it in the next release 💯

**Comment by GeorgeLeithead** [comment]
> Hey Gang! Just checking in - are we good to merge this PR today or tomorrow (Wednesday or Thursday)?
> 
> We plan on pushing a new release to NuGet on Friday and would love to include this PR!
> 
> If it needs more time, no worries! We can add it in the next release 💯

@bijington @brminnick This **specific** PR still has problems with clipping on Windows.  There is another PR #565 which includes a few great fixes, that IS very much good to go.
I would however be happy to have this PR released and a NEW bug/issue raised specifically for the Windows clipping.  The clipping on other platforms appear to be good.

**Comment by brminnick** [comment]
Thanks George! Could you open an Issue specific to the Windows bug?

**Review by brminnick — APPROVED** [review]
Thanks George!

**Pull request merged**
